config.py

```diff
@@ -43,20 +47,24 @@ class Configuration(object):
     
     @classmethod
     def load(cls):
-        cfv = 'SIMPLIFIED_CONFIGURATION_FILE'
-        if not cfv in os.environ:
-            raise CannotLoadConfiguration(
-                "No configuration file defined in %s." % cfv)
+        """Load additional site configuration from a config file.
 
-        config_path = os.environ[cfv]
-        try:
-            cls.log.info("Loading configuration from %s", config_path)
-            configuration = cls._load(open(config_path).read())
-        except Exception, e:
-            raise CannotLoadConfiguration(
-                "Error loading configuration file %s: %s" % (
-                    config_path, e)
-            )
+        This is being phased out in favor of taking all configuration from a
+        database.
```

Comment:
How come we can't get rid of this now?

Reply:
I was just being conservative -- this code is still in the circ manager. I'd need to make sure we change over on the next deployment but since there's only one library registry and it hasn't been deployed in a year I'll need to take special care during the next deployment anyway.